asserts/confdb.go

```diff
@@ -28,33 +28,33 @@ import (
 	"github.com/snapcore/snapd/confdb"
 )
 
-// Confdb holds a confdb assertion, which is a definition by an account of
-// access views and a storage schema for a set of related configuration options
-// under the purview of the account.
-type Confdb struct {
+// ConfdbSchema holds a confdb-schema assertion, which is a definition by an
+// account of access views and a storage schema for a set of related
+// configuration options under the purview of the account.
+type ConfdbSchema struct {
 	assertionBase
 
-	confdb    *confdb.Confdb
-	timestamp time.Time
+	confdbSchema *confdb.Schema
+	timestamp    time.Time
 }
 
 // AccountID returns the identifier of the account that signed this assertion.
-func (ar *Confdb) AccountID() string {
+func (ar *ConfdbSchema) AccountID() string {
 	return ar.HeaderString("account-id")
 }
 
 // Name returns the name for the confdb.
-func (ar *Confdb) Name() string {
+func (ar *ConfdbSchema) Name() string {
 	return ar.HeaderString("name")
 }
 
-// Confdb returns a Confdb assembled from the assertion that can be used
+// ConfdbSchema returns a ConfdbSchema assembled from the assertion that can be used
 // to access confdb views.
-func (ar *Confdb) Confdb() *confdb.Confdb {
-	return ar.confdb
+func (ar *ConfdbSchema) ConfdbSchema() *confdb.Schema {
```

Comment:
this probably can just be called Schema() ?

Reply:
I left this as-is intentionally, there's always the "storage" schema as well so I thought it might be better to leave the prefix

Reply:
but the storage schema lives under the Confdb one?